basic/src/test/java/org/jboss/hal/testsuite/test/runtime/WebMetricsTestCase.java

```diff
@@ -52,7 +52,7 @@
     private CliClient cliClient = CliClientFactory.getClient();
     private FinderNavigation navigation;
     private DefaultContext statementContext;
-    private Dispatcher dispatcher;
+    private static Dispatcher dispatcher;
```

Comment:
Sorry, my mistake. I did not see it after I tried something with `@BeforeClass`.